sample-apps/discounts/web/index.js

```diff
@@ -157,43 +159,40 @@ app.post(
 // All endpoints after this point will require an active session
 app.use("/api/*", shopify.validateAuthenticatedSession());
 
-// All endpoints after this point will have access to a request.body
-// attribute, as a result of the express.json() middleware
 app.use(express.json());
 
-app.get("/api/products/count", async (req, res) => {
-  const countData = await shopify.api.rest.Product.count({
-    session: res.locals.shopify.session,
-  });
-  res.status(200).send(countData);
-});
-
-app.get("/api/products/create", async (req, res) => {
-  let status = 200;
-  let error = null;
-
-  try {
-    await productCreator(res.locals.shopify.session);
-  } catch (e) {
-    status = 500;
-    error = e.message;
+// Helper function for handling any user-facing errors in GraphQL responses
+function handleUserError(userErrors, res) {
```

Comment:
This function doesn't look like it's used in here. We should remove it since the frontend handles the user errors.